src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page.md

```diff
@@ -41,6 +41,7 @@ Page provides methods to interact with a single tab in a running web browser. A
 | [page.locator(selector)](/javascript-api/k6-experimental/browser/page/locator/)       | Returns a [Locator](/javascript-api/k6-experimental/browser/locator/) for the given `selector`.                                          |
 | [page.mainFrame()](/javascript-api/k6-experimental/browser/page/mainframe/)       | Returns the page's main [Frame](/javascript-api/k6-experimental/browser/frame/).                                         |
 | [page.mouse](/javascript-api/k6-experimental/browser/page/mouse/)       | Returns the [Mouse](/javascript-api/k6-experimental/browser/mouse/) instance to interact with a virtual mouse on the page.                                        |
+| [page.on(event, handler)](/javascript-api/k6-experimental/browser/page/on/)       | Accepts a handler function that will be executed every time the specified event is emitted.                                        |
```

Comment:
```suggestion
| [page.on(event, handler)](/javascript-api/k6-experimental/browser/page/on/)       | Registers a handler function that listens for specified page events emitted.                                        |
```

Reply:
In JavaScript, handler functions are called handlers. What about?

```suggestion
| [page.on(event, handler)](/javascript-api/k6-experimental/browser/page/on/)       | Registers a handler to be called whenever the specified event occurs.                                        |
```

Reply:
I like that! Fixed in [6ab6730](https://github.com/grafana/k6-docs/pull/1308/commits/6ab673090236c027f913741a0c0075c3ab48f493).